treeherder/perf/management/commands/import_perf_data.py

```diff
@@ -60,7 +60,7 @@ def _add_series(pc, project_name, signature_hash, signature_props, verbosity):
 
     with transaction.atomic():
         for datum in series:
-            PerformanceDatum.objects.create(
+            PerformanceDatum.objects.get_or_create(
```

Comment:
Since you don't need to retrieve the object created, and the case where you have a duplicate is very uncommon, I would except the integrity error and log it with a warning level. It's not a huge difference, you just save one select query for each insertion with no duplicates. I'll let the final call to you. 
If you want to go with get_or_create you should put `value` in the defaults parameter, so that it doesn't raise an IntegrityError when you insert a duplicate datum with a different value. Again, final call to you since you know the domain better.
:shipit: